### PR TITLE
feat: add account activity

### PR DESCRIPTION
**Pull request by raop155**
# Description

- Add `Account` activity

image
image

### CODE REVIEW

File: apps/web/src/features/accounts/components/account-sections/account-activity.tsx [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+  value: bigint | undefined;
+  nonce: number | undefined;
+  hash: string | undefined;
+  chainId: bigint | undefined;
+  gasLimit: bigint | undefined;
+  timestamp: number | undefined;
+};
+
+const txHashes = [
+  '0x73c1923e1ea513c924ab5a0b121dd919caae89b147023b78e814dfa0a2b59514',
```
Comment (elranu) [inline apps/web/src/features/accounts/components/account-sections/account-activity.tsx]: probably we need to delete this hashes, right?

File: apps/web/src/features/accounts/components/account-sections/account-activity.tsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+  return async (): Promise<TransactionData> => {
+    const provider = new JsonRpcProvider(network);
+    const receipt = await provider.waitForTransaction(txHash);
+    const transaction = await provider.getTransaction(txHash);
+
+    if (!receipt && !transaction) return Promise.reject("Couldn't find transaction");
+
+    const block = await provider.getBlock(transaction?.blockNumber!);
+
+    // console.log({
```
Comment (elranu) [inline apps/web/src/features/accounts/components/account-sections/account-activity.tsx]: remove

File: apps/web/src/features/accounts/components/send-dialog.tsx [hunk 3]
```diff
@@ -17,11 +17,16 @@ import { useNetwork } from '@/shared/hooks/use-network';
 import { formatBalance } from '@/shared/utils/crypto';
 import { parseEther } from 'ethers';
 import { Send } from 'lucide-react';
-import { FormEvent } from 'react';
+import { FormEvent, useState } from 'react';
+import { setTxHash } from '../utils/transactions';
 
 const SendDialog = ({ account }: { account: WalletInfo | null }) => {
+  // TODO: This only works for Mumbai testnet <<
```
Comment (elranu) [inline apps/web/src/features/accounts/components/send-dialog.tsx]: is true this TODO?

Comment (raop155) [reply apps/web/src/features/accounts/components/send-dialog.tsx]: Change it to make it dynamic

**Pull request merged**